src/main/java/johnny/Johnny.java

```diff
@@ -1,5 +1,19 @@
 package johnny;
 
+import javafx.application.Application;
```

Comment:
Overall very well written and well designed. Hard to find substantial errors.

Reply:
Thanks for being so detailed